### DISCUSSION TITLE
Using DateTimeOffset instead of DateTime

### DISCUSSION THREAD

**Discussion by mastermehdi** [DaveSkender/Stock.Indicators#746] [Q&A] [answered]
Isn't it better to convert all "DateTime" types to "DateTimeOffset" types?

**Comment by DaveSkender** [comment, marked as answer]
Time is relative.  Some smart dude said that.

`DateTimeOffset` is a more complex `struct`, so it is a slightly higher burden to carry and would be good in multi-localization in/out use cases.  `DateTime` can still carry time zone info and can be standardized on UTC or a local time and shown/converted when needed.

I think `DateTime` is better for this use case, but happy to debate on this if others have opinions.

Ref: https://docs.microsoft.com/dotnet/standard/datetime/choosing-between-datetime
